src/routes/companies/companies.controller.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../../db");
+
+const companiesController = (req, res) => {
+    db.collection('companies')
+    .get()
+    .then(snapshot => snapshot.docs.map(doc => doc.data()))
+    .then(data=>{
+        res.json({
+            data
+        })
+    })
```

Comment:
Good job, It might be worth adding a `.catch` error so in case there is an error, the API does not crash but just logs the error? For example,
```suggestion
    }).catch(error => console.log('companies data error', error));
```